_delphi_utils_python/delphi_utils/geomap.py

```diff
@@ -596,50 +594,43 @@ def get_geo_values(self, geo_type):
             - all counties within a state
             - all states within an hhs region
 
         Parameters
         ----------
-        container_geocode: str of identity of nation/state/hhs
-            "fips" for return container_geocode of state
-            "state_id" for return container_geocode of nation and hhs
+        container_geocode: str
+            Instance of nation/state/hhs to find the sub-regions of
         contained_geocode_type: str
-            One of "state","county"
+            The subregion type to get all instance of. One of "state", "county"
         container_geocode_type: str
-            One of "state","nation","hhs"
+            The parent region type. One of "state", "nation", "hhs"
 
         Returns
         -------
-        Set of geo ids of the specified type that lie within the specified container geocode,
-        all in string format.
+        Set of geo code strings of the given type that lie within the given container geocode.
         """
-        if contained_geocode_type not in ("county","state"):
-            raise ValueError("contained_geocode_type must be one of state, nation and hhs")
-        geo_values=set()
-        if contained_geocode_type=="state":
-            if container_geocode_type=="nation" and container_geocode=="us":
+        if contained_geocode_type == "state":
+            if container_geocode_type == "nation" and container_geocode == "us":
                 crosswalk = self._load_crosswalk_from_file("state", "state")
-                geo_values=set(crosswalk["state_id"])
-            if container_geocode_type=="hhs":
+                return set(crosswalk["state_id"])
+            if container_geocode_type == "hhs":
                 crosswalk_hhs = self._load_crosswalk_from_file("fips", "hhs")
                 crosswalk_state = self._load_crosswalk_from_file("fips", "state")
-                fips_hhs=crosswalk_hhs[crosswalk_hhs["hhs"] == container_geocode]["fips"]
-                fips_state_id=crosswalk_state[crosswalk_state["fips"].isin(fips_hhs)]["state_id"]
-                geo_values.update(fips_state_id)
-        elif contained_geocode_type=="county" and container_geocode_type=="state":
+                fips_hhs = crosswalk_hhs[crosswalk_hhs["hhs"] == container_geocode]["fips"]
+                return set(crosswalk_state[crosswalk_state["fips"].isin(fips_hhs)]["state_id"])
+        elif contained_geocode_type == "county" and container_geocode_type == "state":
             crosswalk = self._load_crosswalk_from_file("fips", "state")
-            geo_values=crosswalk[crosswalk["state_id"]==container_geocode]["fips"]
-        else:
-            raise ValueError("Do not satisfied the reqirement")
-        return geo_values
+            return set(crosswalk[crosswalk["state_id"] == container_geocode]["fips"])
+        raise ValueError("(contained_geocode_type, container_geocode_type) must be one of "
+                         "(state, nation), (state, hhs), (county, state)")
```

Comment:
Include `f"({contained_geocode_type}, {container_geocode_type})"` in this message to give future encounters a leg up

Reply:
Added this into the message!